router/thundercat-router.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const Thundercat = require('../model/thundercat.js');
+const createError = require('http-errors');
+const debug = require('debug')('thundercatapp:thundercat-router');
+
+
+const thundercatRouter = module.exports = new Router;
+
+
+thundercatRouter.post('/api/thundercats',jsonParser, function(req, res, next) {
+  debug('POST /api/thundercats');
+  new Thundercat(req.body).save()
+  .then(thundercat => res.json(thundercat))
+  .catch(next);
+});
+
+thundercatRouter.get('/api/thundercats/:id', function(req, res, next){
+  debug('GET /api/thundercats/:id');
+  Thundercat.findById(req.params.id)
+  .then(thundercat => res.json(thundercat))
+  .catch(err => next(createError(404, err.message)));
+});
+
+thundercatRouter.get('/api/thundercats', function(req, res, next){
```

Comment:
This GET route to get all IDs is no longer working since we're using Mongo. There is no more ``Thundercat.fetchAll`` method.